### PR TITLE
Updated conda activate command

### PR DESCRIPTION
**Pull request by pvskand**
Updated conda activate command (for the newer versions of conda >= 4.4) [As discussed on Gitter with @lilleswing]

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -64,7 +64,8 @@ python setup.py install                                # Manual install
 nosetests -a '!slow' -v deepchem --nologcapture        # Run tests
 ```
 This creates a new conda environment `deepchem` and installs in it the dependencies that
-are needed. To access it, use the `source activate deepchem` command.
+are needed. To access it, use the `conda activate deepchem` command (if your conda version >= 4.4) and use `source activate deepchem` command (if your conda version < 4.4).
```
Comment (lilleswing) [inline README.md]: I think we should simplify and assume users are on conda >= 4.4.

Comment (pvskand) [reply README.md]: I wrote both the commands since some of them might be in a lesser version and would be stuck at this command. What do you say?

Comment (peastman) [reply README.md]: It's not a very old version.  4.4 was just released in December.  And there sometimes are good reasons someone can't or doesn't want to use the very latest version of something.

Comment (lilleswing) [reply README.md]: Ok lets use the statement as in this PR then.

Comment (pvskand) [reply README.md]: Okay

**Comment by rbharath** [comment]
LGTM

**Pull request merged**